Android Crash: android.os.DeadObjectException

From Play Console:
```
java.lang.RuntimeException: 
  at android.app.ActivityThread.handleStopService (ActivityThread.java:4369)
  at android.app.ActivityThread.access$1900 (ActivityThread.java:237)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1956)
  at android.os.Handler.dispatchMessage (Handler.java:106)
  at android.os.Looper.loop (Looper.java:223)
  at android.app.ActivityThread.main (ActivityThread.java:7660)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:592)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:947)
Caused by: android.os.DeadObjectException: 
  at android.os.BinderProxy.transactNative (Native Method)
  at android.os.BinderProxy.transact (BinderProxy.java:550)
  at org.mozilla.firefox.vpn.VPNServiceBinder.dispatchEvent (VPNServiceBinder.kt:181)
  at com.mozilla.vpn.VPNTunnel.onStateChange (VPNTunnel.kt:33)
  at com.wireguard.android.backend.GoBackend$VpnService.onDestroy (GoBackend.java:370)
  at android.app.ActivityThread.handleStopService (ActivityThread.java:4349)

```

--- a/android/src/com/mozilla/vpn/VPNServiceBinder.kt
+++ b/android/src/com/mozilla/vpn/VPNServiceBinder.kt
@@ -5,6 +5,7 @@
 package org.mozilla.firefox.vpn
 import android.content.Context
 import android.os.Binder
+import android.os.DeadObjectException
 import android.os.IBinder
 import android.os.Parcel
 import android.util.Log
@@ -12,10 +13,10 @@ import com.mozilla.vpn.NotificationUtil
 import com.wireguard.android.backend.Tunnel
 import com.wireguard.config.*
 import com.wireguard.crypto.Key
+import org.json.JSONObject
 import java.io.BufferedReader
 import java.io.InputStreamReader
 import java.lang.Exception
-import org.json.JSONObject
 
 class VPNServiceBinder(service: VPNService) : Binder() {
 
@@ -174,12 +175,17 @@ class VPNServiceBinder(service: VPNService) : Binder() {
      * [ACTIONS.registerEventListener]
      */
     fun dispatchEvent(code: Int, payload: String) {
-        mListener?.let {
-            if (it.isBinderAlive) {
-                val data = Parcel.obtain()
-                data.writeByteArray(payload.toByteArray(charset("UTF-8")))
-                it.transact(code, data, Parcel.obtain(), 0)
+        try {
+            mListener?.let {
+                if (it.isBinderAlive) {
+                    val data = Parcel.obtain()
+                    data.writeByteArray(payload.toByteArray(charset("UTF-8")))
+                    it.transact(code, data, Parcel.obtain(), 0)
+                }
             }
+        } catch (e: DeadObjectException) {
+            // If the QT Process is killed (not just inactive)
+            // we cant access isBinderAlive, so nothing to do here.
         }
     }
 

